Musical Collection/Musical Collection Console App/Classes/Playlist.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Musical_Collection_Console_App.Interfaces.Classes_Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Musical_Collection_Console_App.Classes
+{
+    public class Playlist : MusicalCollection, IPlaylist 
+    {
+        private string creator;
+
+        public Playlist(string name) : base(name)
+        {
```

Comment:
When you are passing arguments to base constructor move it on the new line .
Also no need for line break in empty braces